overlord/devicestate/handlers_install.go

```diff
@@ -55,9 +56,30 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 	}
 	gadgetDir := gadgetInfo.MountDir()
 
+	args := []string{
+		// create partitions missing from the device
+		"create-partitions",
+		// mount filesystems after they're created
+		"--mount",
+	}
+
+	useEncryption, err := checkEncryption(deviceCtx.Model())
+	if err != nil {
+		return err
+	}
+	if useEncryption {
+		args = append(args,
+			// enable data encryption
+			"--encrypt",
+			// location to store the sealed keyfile
+			"--keyfile", filepath.Join(dirs.RunMnt, "ubuntu-boot", "keyfile"),
```

Comment:
snap-bootstrap expects "--key-file"

Reply:
Fixed. `--key-file` is consistent with cryptsetup.